### PR TITLE
Initialized Project: Created folder structure for server and routes, …

### PR DESCRIPTION
**Pull request by siratl**
…installed dependencies and updated Readme with Schema template

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,591 @@
...
+
+##### 201 (created)
+
+###### Example Response
+
+```
+  {
+    "id": 1,
+    "habitTitle": "Run 10 miles",
+    "category": "Physical Fitness",
```
Comment (theryanclausen) [inline README.md:244]: I think category should have it's own table and be connected in via foreign key. Front end has some places where they need to access all habits by category

File: README.md [hunk 2]
```diff
@@ -1,5 +1,591 @@
...
+    "completed": false,
+    "userId": 2
```
Comment (theryanclausen) [inline README.md:246]: we also need a way to track the completion history for each task

**Review by theryanclausen — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (siratl) [reply README.md:244]: Yes I was going to update that after I create the database, I just wanted to get some info out there that match today's milestone

File: README.md (hunk 2 above)
Comment (siratl) [reply README.md:246]: I'll add that shortly

**Pull request merged**